liota/dccs/iotcc.py

```diff
@@ -71,7 +71,6 @@ def __init__(self, con):
         thread.start()
         # Wait for Subscription to be complete and then proceed to publish message
         time.sleep(0.5)
-        self.proto = HelixProtocol(self.comms, self.comms.identity.username, self.comms.identity.password)
```

Comment:
If we never do this how does username & password get set and used? Oh, right. We're bundled in the MQTT session now which is authenticated and secured.

Reply:
Yes, you are right.